src/main/java/com/cloudera/sparkts/api/java/DateTimeIndexFactory.java

```diff
@@ -110,6 +110,20 @@ public static IrregularDateTimeIndex irregular(long[] dts, ZoneId zone) {
     }
 
     /**
+     * Create a HybridDateTimeIndex composed of the given indices using the default date-time zone.
```

Comment:
It seems like the behavior could be unexpected here if indices without the default zone are passed in, but the default date-time zone is used.  Is that in fact what would happen here?  It might make more sense to require that all indices have the same zone, and then just offer a single instantiator?